package.json

```diff
@@ -18,7 +18,7 @@
   "devDependencies": {
     "@testing-library/cypress": "^8.0.0",
     "@types/uuid": "3.4.7",
-    "cypress": "^8.7.0",
+    "cypress": "^10.4.0",
```

Comment:
We should have either a `yarn.lock` or `package-lock.json` file in this repo. I'd strongly prefer using Yarn, since that's what we use for the other 3.x repos.

Reply:
Fixed with [4e45b27](https://github.com/openmrs/openmrs-test-3refapp/pull/10/commits/4e45b2769527074ed41ff1cc693000fdf6805a29)